schema-guide.md

```diff
@@ -959,16 +967,16 @@ authors:
 - **usage**:<br><br>
     ```yaml
     authors:
-      - name: "The Research Software Project"
-        address: "742 Evergreen Terrace"
-        alias: "RSP"
-        city: "Berlin"
-        country: "DE"
-        date-end: 2018-07-27
-        date-start: 2021-07-27
-        email: "team@research-project.org"
+      - address: "742 Evergreen Terrace"
+        alias: RSP
+        city: Berlin
+        country: DE
+        date-end: "2018-07-27"
+        date-start: "2021-07-27"
+        email: team@research-project.org
```

Comment:
shouldnt email addresses be quoted? yamllint says no, but your bulletlist at the top says yes

Reply:
`@` is a reserved indicator in YAML and can't start a plain scalar.
So email addresses are safe to leave unquoted.
However, this should be addressed in the docs I guess, I'll see if I can do this in a non-distruptive way.

Reply:
Resolved in cb83280.